server/athenian/api/openapi/openapi.yaml

```diff
@@ -3083,11 +3106,25 @@ components:
         stage_timings:
           $ref: '#/components/schemas/StageTimings'
         properties:
-          description: List of the PR properties triggered in the given time window.
+          description: DEPRECATED. List of the PR properties triggered in the given time window.
             There may be extra properties that were not originally requested.
           items:
             $ref: '#/components/schemas/PullRequestProperty'
           type: array
+        events_time_machine:
+          description: List of PR events which happened until `date_to`.
+          items:
+            $ref: '#/components/schemas/PullRequestEvent'
+          type: array
+        stage_time_machine:
+          $ref: '#/components/schemas/PullRequestStage'
```

Comment:
I miss the `description` field

Reply:
Me too, but it is against OpenAPI spec when you `$ref` :disappointed:

Reply:
Don't worry, I'll describe the events and stages after getting approval on the general structure.